rasa/nlu/model.py

```diff
@@ -289,11 +289,10 @@ def ensure_model_compatibility(
         model_version = metadata.get("rasa_version", "0.0.0")
         if version.parse(model_version) < version.parse(version_to_check):
             raise UnsupportedModelError(
-                "The model version is too old to be "
-                "loaded by this Rasa NLU instance. "
-                "Either retrain the model, or run with "
-                "an older version. "
-                "Model version: {} Instance version: {}"
+                "The model version is trained using Rasa {}"
+                "and is not compatible with your current installation ({}). "
+                "This means that you probably want to retrain your model "
```

Comment:
optional: No strong opinion on this, but I don't think we should give instructions using words like `probably`. What do you think? 
```suggestion
                "This means that you either need to retrain your model "
```